services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/unit_templates_manager/unit_template_minis.jsx

```diff
@@ -102,8 +102,24 @@ export default class UnitTemplateMinis extends React.Component {
     )
   };
 
+  getModelCardsByType(models, type) {
+    const filteredModels = models.filter(model => {
+      const { unit_template_category, id } = model
+      if(id === CREATE_YOUR_OWN_ID) {
+        // we want to include the create your own pack card as the last card in the last section which is Whole Class Lessons
+        if(type !== WHOLE_CLASS_LESSONS) { return }
+        return model
+      }
+      if(type === LANGUAGE_SKILLS) {
+        return ACTIVITY_PACK_TYPES[2].types.includes(unit_template_category.name)
```

Comment:
the `ACTIVITY_PACK_TYPES[2]` strikes me as subject to change here... is there another way to identify that `type` (like a `.find`)?